tox.ini

```diff
@@ -33,37 +33,35 @@ deps =
     pymssql; platform_system=="Linux" # we only test this on Linux
     latest: https://github.com/biolab/orange-canvas-core/archive/refs/heads/master.zip#egg=orange-canvas-core
     latest: https://github.com/biolab/orange-widget-base/archive/refs/heads/master.zip#egg=orange-widget-base
-    # pip is not in the requirements; here to see what happens with an old pip version
-    oldest: pip==20.3
     # GUI requirements
     oldest: orange-canvas-core==0.2.5
     oldest: orange-widget-base==4.25.0
     oldest: AnyQt==0.2.0
-    oldest: matplotlib==3.2.0
+    oldest: matplotlib==3.6.0
     oldest: pygments==2.8.0
     oldest: pyqtgraph>=0.13.1
     oldest: qtconsole==4.7.2
     # core requirements
     oldest: baycomp==1.0.2
-    oldest: bottleneck==1.3.4
-    oldest: catboost==1.0.1
+    oldest: bottleneck==1.3.7
+    oldest: catboost==1.2.2
     oldest: chardet==3.0.2
     oldest: httpx==0.21.0
     oldest: joblib==1.2.0
     # oldest: keyring
     # oldest: keyrings.alt
     # oldest: networkx
-    oldest: numpy==1.21
+    oldest: numpy==1.23.2
     oldest: openpyxl==3.1.3
-    oldest: openTSNE==0.6.2
-    oldest: pandas==1.4.0
+    oldest: openTSNE==1.0.0
+    oldest: pandas==1.5.1
     oldest: python-louvain==0.13
     # oldest: pyyaml
     # oldest: requests
     oldest: scikit-learn==1.5.1
-    oldest: scipy==1.9
+    oldest: scipy==1.10
     # oldest: serverfiles
-    oldest: xgboost==1.7.4
+    oldest: xgboost==2.1.0
```

Comment:
Why did you need to update xgboost? The older version does not work with python 3.11 anmore on linux?

And the rest, were they lifted for compatibility with python 3.11?

Should we also update requirements accordingly?

Reply:
> Why did you need to update xgboost? The older version does not work with python 3.11 anmore on linux?

tox fails in `pip check` step with: `xgboost 1.7.4 is not supported on this platform`.



> And the rest, were they lifted for compatibility with python 3.11?

Mostly oldest available wheels for 3.11